assets/sass/components/_navigation.scss

```diff
@@ -152,11 +152,14 @@
         }
     }
     .split-nav__prev {
+        margin-right: $spacingUnit;
```

Comment:
This fixes a display issue where these elements could butt up against one another.